### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by ahawkins94**
Added carousel, tidied up some code and begun playing around with a background. Also sorted out the login/sign up screens

### CODE REVIEW

**Review by CDacres — APPROVED** [review]
Wow, there's been a lot of cleanup!

Looks ok to me, we just need to resolve the merge conflicts in browser.

**Comment by HenrySpartGlobal** [comment]
Okay, I think we're good to go after solving that merge conflict

**Comment by CDacres** [comment]
Great! So we're ready to merge.

**Pull request merged**
